### PR TITLE
Add Libravatar support / Miscellaneous

### PR DESCRIPTION
**Pull request by baseplate-admin**
[**GitLab repo**](https://git.linux-kernel.at/oliver/ivatar/)

# **Websites :**

- https://www.libravatar.org/

# **Examples :**
- https://pagure.io

### CODE REVIEW

File: src/technologies/l.json [hunk 1]
```diff
@@ -11,6 +11,15 @@
     },
     "website": "http://www.lepton-cms.org"
   },
+  "Libravatar": {
+    "cats": [
+      19
+    ],
+    "description": "freeing the web one face at a time.",
```
Comment (nurbek91) [inline src/technologies/l.json:18]: "description": "Libravatar is a protocol specification for fetching avatar images for e-mail addresses and OpenIDs in a decentralized way.",

File: src/technologies/l.json [hunk 2]
```diff
@@ -11,6 +11,15 @@
...
+    "html": "<[^>]+libravatar\\.org/avatar/",
```
Comment (nurbek91) [inline src/technologies/l.json:19]: lets use dom instead html:
example: https://pagure.io
`"dom": "img[src*='.libravatar.org/avatar/']",`
`"oss": true,`

**Review by nurbek91 — COMMENTED** [review]
Hey there, thanks for PR! Little changes required.

File: src/technologies/l.json (hunk 1 above)
Comment (baseplate-admin) [reply src/technologies/l.json:18]: Hi there, 
Thanks for taking a look. 😄
But afaik Libravatar is not decentralised.( its a service hosted in AWS ) 

More = https://blog.libravatar.org/posts/Back_To_Fedora/

But if you wish i can add the description anyways.

File: src/technologies/l.json (hunk 2 above)
Comment (baseplate-admin) [reply src/technologies/l.json]: Done. Thank you very much for pointing this out.

File: src/technologies/l.json (hunk 1 above)
Comment (nurbek91) [reply src/technologies/l.json:18]: Your last desc perfect fit. The main idea was to start the description with:
Libravatar is a ....

Comment (baseplate-admin) [reply src/technologies/l.json:18]: Thank you very much for accepting it 😄

**Comment by baseplate-admin** [comment]
Thank you again @nurbek91 For pointing my mistakes out.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**